iOS/Account/FeedWranglerAccountViewController.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  FeedWranglerAccountViewController.swift
+//  NetNewsWire
+//
+//  Created by Jonathan Bennett on 2019-11-24.
+//  Copyright © 2019 Ranchero Software. All rights reserved.
+//
+
+import UIKit
+import Account
+import RSWeb
+
+class FeedWranglerAccountViewController: UITableViewController {
+
+	@IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+	@IBOutlet weak var cancelBarButtonItem: UIBarButtonItem!
+	@IBOutlet weak var emailTextField: UITextField!
+	@IBOutlet weak var passwordTextField: UITextField!
+	@IBOutlet weak var showHideButton: UIButton!
+	@IBOutlet weak var actionButton: UIButton!
+	
+	weak var account: Account?
+	weak var delegate: AddAccountDismissDelegate?
+
+	override func viewDidLoad() {
+        super.viewDidLoad()
+
+		activityIndicator.isHidden = true
+		emailTextField.delegate = self
+		passwordTextField.delegate = self
+		
+		if let account = account, let credentials = try? account.retrieveCredentials(type: .feedWranglerBasic) {
+			actionButton.setTitle(NSLocalizedString("Update Credentials", comment: "Update Credentials"), for: .normal)
+			emailTextField.text = credentials.username
+			passwordTextField.text = credentials.secret
+		} else {
+			actionButton.setTitle(NSLocalizedString("Add Account", comment: "Update Credentials"), for: .normal)
+		}
+
+		NotificationCenter.default.addObserver(self, selector: #selector(textDidChange(_:)), name: UITextField.textDidChangeNotification, object: emailTextField)
+		NotificationCenter.default.addObserver(self, selector: #selector(textDidChange(_:)), name: UITextField.textDidChangeNotification, object: passwordTextField)
+
+		tableView.register(ImageHeaderView.self, forHeaderFooterViewReuseIdentifier: "SectionHeader")
+	}
+
+	override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+		return section == 0 ? ImageHeaderView.rowHeight : super.tableView(tableView, heightForHeaderInSection: section)
+	}
+	
+	override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+		if section == 0 {
+			let headerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: "SectionHeader") as! ImageHeaderView
+			headerView.imageView.image = AppAssets.image(for: .feedWrangler)
+			return headerView
+		} else {
+			return super.tableView(tableView, viewForHeaderInSection: section)
+		}
+	}
+
+	@IBAction func cancel(_ sender: Any) {
+		dismiss(animated: true, completion: nil)
+		delegate?.dismiss()
+	}
+	
+	@IBAction func showHidePassword(_ sender: Any) {
+		if passwordTextField.isSecureTextEntry {
+			passwordTextField.isSecureTextEntry = false
+			showHideButton.setTitle("Hide", for: .normal)
+		} else {
+			passwordTextField.isSecureTextEntry = true
+			showHideButton.setTitle("Show", for: .normal)
+		}
+	}
```

Comment:
I thought you might like to know I noticed you're using `NSLocalizedString` in every other place except here.